homeassistant/components/denonavr/media_player.py

```diff
@@ -237,6 +150,18 @@ def update(self):
             self._sound_mode = self._receiver.sound_mode
             self._sound_mode_raw = self._receiver.sound_mode_raw
 
+    @property
+    def unique_id(self):
+        """Return the unique id of the zone."""
+        return self._unique_id
+
+    @property
+    def device_info(self):
+        """Return the device info of the receiver."""
+        return {
+            "identifiers": {(DOMAIN, self._receiver_device_id)},
+        }
+
```

Comment:
I was hoping this could be removed, but there is no way to link the entity to the device without it

Reply:
Yes that is needed to link the device to the entity indeed